napari/_qt/experimental/render/qt_test_image.py

```diff
@@ -2,8 +2,14 @@
 """
 from typing import Callable, Tuple
 
-import numpy as np
-from qtpy.QtWidgets import QCheckBox, QFrame, QPushButton, QVBoxLayout
+from qtpy.QtWidgets import (
+    QCheckBox,
+    QComboBox,
+    QFrame,
+    QPushButton,
+    QVBoxLayout,
+)
+from skimage import data as skimage_data
```

Comment:
@pwinston i just realized, the main repo does not actually depend on scikit-image, we have it in our test dependencies, so tests will pass with this, but it's not actually required. Maybe you can make the optional in #1834, and my mistake for missing this in review!